### PR TITLE
[18.0][MIG] sale_force_invoiced

### PR DESCRIPTION
**Pull request by odooexperts-visioncode**

### CODE REVIEW

File: sale_force_invoiced/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Sale Force Invoiced",
     "summary": "Allows to force the invoice status of the sales order to Invoiced",
-    "version": "17.0.1.1.1",
+    "version": "18.0.1.0.0",
```
Comment (rousseldenis) [inline sale_force_invoiced/__manifest__.py:7]: this change should be in migration commit

**Comment by rousseldenis** [comment]
@odooexperts-visioncode Ok, I think you misunderstood what I wanted to say.

You should follow this : 
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0#technical-method-to-migrate-a-module-from-170-to-180-branch

The important part on the line with pre-commit stuff is the `--no-verify` option. Unless it will fail and ask you to change the version number. Pre-commit changes should stay in separate commit.

Then, you update the module version  (and other changes maybe) and then you put them in the migration commit.

**Comment by odooexperts-visioncode** [comment]
@rousseldenis Currently, there are not any changes in the pre-commit run, only pylint Failure for the version.

image

As there are not any change in the pre-commit, I might commit with --allow-empty

So, should I push an empty commit initially or the Direct Migration commit will be okay?

**Comment by rousseldenis** [comment]
> @rousseldenis Currently, there are not any changes in the pre-commit run, only pylint Failure for the version.
> 
> image
> 
> As there are not any change in the pre-commit, I might commit with --allow-empty
> 
> So, should I push an empty commit initially or the Direct Migration commit will be okay?

As I said, follow the procedure after running `pre-commit run -a`, do `git commit -m "[IMP] $module: pre-commit auto fixes"  --no-verify`


image

**Comment by rousseldenis** [comment]
> As there are not any change in the pre-commit, I might commit with --allow-empty

Ooooh, ok. So, it's ok as is

**Comment by odooexperts-visioncode** [comment]
@rousseldenis So, do I push 2 commits (1 empty and 1 migration) or just a single migration commit is okay?

**Review by sheragar — APPROVED** [review]
LGTM. Funtional Review 👍

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

**Comment by ploegvde** [comment]
@pedrobaeza Hi, you probably know. If I want to translate and the project is not there on: https://translation.odoo-community.org/projects/sale-workflow-18-0/sale-workflow-18-0-sale_force_invoiced/

What to do? Tnx!

**Comment by pedrobaeza** [comment]
Yes, it is:

imagen


Have you logged in?

**Comment by ploegvde** [comment]
@pedrobaeza Yes logged in, but think I have to be added somewhere. Not translated before on this platform. 
image

**Comment by pedrobaeza** [comment]
Then yes, you have to write to get access:

https://odoo-community.org/resources/translate

**Comment by ploegvde** [comment]
Oops. Have to read better. I have sent the mail. Gracias!

**Comment by ploegvde** [comment]
@pedrobaeza Sorry hopefully last mail. How long will it take after the mail? Nothing happened so far.

**Comment by pedrobaeza** [comment]
It's processed manually. You have to wait a bit.

**Pull request merged**